storage/bag_replicator/src/test/scala/uk/ac/wellcome/platform/archive/bagreplicator/BagStorageTest.scala

```diff
@@ -26,33 +24,50 @@ class BagStorageTest
     with IntegrationPatience
     with BagReplicatorFixtures {
 
-  it("should duplicate a bag into a given location") {
+  val bagStorage = new BagStorage(s3Client = s3Client)
```

Comment:
`withBagStorage`?

Reply:
I considered it, but it doesn’t have any state and keeps the indentation cleaner. So I took it out, but I don't feel strongly about it.